src/software/world/field.h

```diff
@@ -35,6 +35,13 @@ typedef enum
 class Field
 {
    public:
+    /**
+     * Creates a field with the standard SSL Division B dimensions
+     *
+     * @return a field with the standard SSL Division B dimensions
+     */
+    static Field createSSLDivisionBField();
```

Comment:
Replace `createSSLDivisionBField` in test_util/test_util.h with this function and remove it from test_util/test_util.h

Reply:
yep, wasn't sure if I wanted to do it in this PR or not though.

Reply:
I created #1400 to do right after this PR, since there's 100+ matches so I think it's better to keep separate.

Reply:
since you're not using it in this PR maybe it's best to put it in another PR

Reply:
yup, removed from this PR and updated #1400